test/Diagnostics/diagnostic_fields_test.jl

```diff
@@ -89,8 +89,7 @@ function config_risingbubble(FT, N, resolution, xmax, ymax, zmax)
 
     # Set up the model
     C_smag = FT(0.23)
-    ref_state =
-        HydrostaticState(DryAdiabaticProfile(typemin(FT), FT(300)), FT(0))
+    ref_state = HydrostaticState(DryAdiabaticProfile(FT(300)), FT(0))
```

Comment:
Looks like DryAdiabaticProfile only has one argument now, so FT(0) I'd think can be omitted as a second argument.

Reply:
`DryAdiabaticProfile` does have one argument, what looks like the second argument is actually the second argument for `HydrostaticState`.

Reply:
I'll move these to separate lines for clarity